niceman/support/sshconnector2.py

```diff
@@ -21,6 +21,16 @@ class SSHConnector2(object):
     Manage and use an SSH connection.
     """
 
+    def connect(self):
```

Comment:
Since we are using the SSH client from starcluster for the PTY connections, we can also use it for executing shell commands on the remote server and drop the use of 2 different SSH libraries by eliminating the sshconnector2.py package.